thunder.install

```diff
@@ -314,21 +313,16 @@ function thunder_update_8102() {
  */
 function thunder_update_8103() {
 
-  /** @var \Drupal\thunder\ThunderUpdateLogger $updateLogger */
-  $updateLogger = Drupal::service('logger.thunder_update');
-  $updateLogger->cleanLogs();
-
   $moduleInstaller = \Drupal::service('module_installer');
 
-  try {
-    if (!$moduleInstaller->install(['thunder_updater'])) {
-      $updateLogger->warning(t('Unable to enable the Thunder Updater'));
-    }
-  }
-  catch (MissingDependencyException $e) {
-    $updateLogger->warning(t('Unable to enable the Thunder Updater because of missing dependencies.'));
+  if (!$moduleInstaller->install(['thunder_updater'])) {
```

Comment:
```ModuleInstaller::install``` will return TRUE or throw (there is no FALSE). So it's possible just to encapsulate everything in try/catch and throw exception -> if you want to provide more descriptive message to user. Or just to let exception from install to bubble up.

Also exceptions should not be translated.